legal-api/src/legal_api/core/filing.py

```diff
@@ -568,6 +572,7 @@ def get_document_list(business,  # pylint: disable=too-many-locals disable=too-m
                     Filing.FilingTypes.CONTINUATIONOUT.value,
                     Filing.FilingTypes.AGMEXTENSION.value,
                     Filing.FilingTypes.AGMLOCATIONCHANGE.value,
+                    Filing.FilingTypes.TRANSPARENCY_REGISTER.value,
```

Comment:
You might need to add this in `no_legal_filings_in_paid_withdrawn_status` as well

Reply:
Good point, will update